pkg/policy/execute.go

```diff
@@ -102,11 +105,33 @@ func NewExecutor(conn *pgxpool.Conn, log hclog.Logger, progressUpdate UpdateCall
 		conn:           conn,
 		log:            log,
 		progressUpdate: progressUpdate,
+		PolicyPath:     &PolicyPath{policyPath: []string{}},
```

Comment:
you want to hold this as a string array []string or even a string. see suggestion in with function